include/deal.II/grid/manifold_lib.h

```diff
@@ -266,6 +266,22 @@ class FunctionManifold : public ChartManifold<dim, spacedim, chartdim>
   push_forward(const Point<chartdim> &chart_point) const;
 
   /**
+   * Given a point in the chartdim dimensional Euclidean space, this
+   * method returns the derivatives of the function $F$ that maps from
+   * the sub_manifold coordinate system to the Euclidean coordinate
+   * system. In other words, it is a matrix of size
+   * $\text{spacedim}\times\text{chartdim}$.
+   *
+   * This function is used in the computations required by the
+   * get_tangent_vector() function.
+   *
+   * Refer to the general documentation of this class for more information.
```

Comment:
This is just the generic blurb. I think it would be useful to say here that it will be computed using finite differencing of the function expression this class is based upon.